audformat/core/utils.py

```diff
@@ -376,6 +360,49 @@ def join_labels(
     return joined_labels
 
 
+def join_schemes(
+        db1: Database,
```

Comment:
Since update supports a list of `Database` objects, shouldn't we also allow the same here?

Reply:
And would it make sense to actually use the new function in `Database.update()` then?

Reply:
The difference is that `update()` is already assigned to a database.
But in principle you are right, we could replace `db1` and `db2` by a single list `dbs` that can contain one or more databases.
Was that your idea?

Reply:
Ah sorry, forget my last comment. We decided earlier that it's safer to let the user do it because of the side-effects it can have.

Reply:
> Was that your idea?

yes

Reply:
Then it's also in line with `join_labels()`.

Reply:
I changed to a sequence of databases.